package.json

```diff
@@ -1,11 +1,42 @@
 {
-  "name": "react-native-keychain",
-  "version": "3.0.0",
+  "_args": [
```

Comment:
I think the changes to package.json can go. This looks like a whole bunch of stuff that gets added when the package is installed